package fr.ruche.exception;

public class UserNotFoundException extends RuntimeException {

}
